examples/cross-contract-calls/Cargo.toml

```diff
@@ -11,11 +11,15 @@ near-units = "0.2.0"
 serde_json = "1.0"
 test-case = "2.0"
 tokio = { version = "1.14", features = ["full"] }
-workspaces = "0.7.0"
+near-workspaces = { version = "0.8.0" }
 
 cross-contract-high-level = { path = "./high-level" }
 cross-contract-low-level = { path = "./low-level" }
 
+
+[patch.crates-io]
+near-sdk = { path = "../../near-sdk" }
```

Comment:
Line 9 probably also has to be changed to `version` to follow the pattern

Reply:
done